doc/build.md

```diff
@@ -29,7 +29,7 @@ Istio Manager requires Go1.8+ toolchain.
 
 Bazel build environment is compatible with the standard Golang tooling, except you need to vendorize all dependencies in Istio Manager. If you have successfully built with Bazel, run the following script to put dependencies fetched by Bazel into `vendor` directory:
 
-    bin/init.sh
+    toolbox/scripts/init.sh
```

Comment:
If you linked another repo, how do you ensure that the script changes there remain consistent with local copy? Isn't this going to be another nightmare to debug the tooling scripts itself ? 

And why do we need this refactoring? Is there any other repository currently using the same set of scripts?